src/main/kotlin/eu/europa/ec/eudi/prex/PresentationMatcher.kt

```diff
@@ -89,7 +89,7 @@ typealias ClaimId = String
 
 interface Claim {
     val uniqueId: ClaimId
-    val format: ClaimFormat
+    val format: String
```

Comment:
Format value class should be used here as format attribute is not any string but a string that can be converted to JsonObject

Reply:
`ClaimFormat` used to be an enumeration. So I think in this case it's ok to replace it by a String